CHANGELOG.md

```diff
@@ -1,5 +1,427 @@
+[#14683]: https://github.com/crystal-lang/crystal/pull/14683
+[#14723]: https://github.com/crystal-lang/crystal/pull/14723
+[#14724]: https://github.com/crystal-lang/crystal/pull/14724
+
+#### compiler
+
+- *(codegen)* Add `Program#size_t` and `Target#size_bit_width` ([#14442], thanks @ysbaddaden)
+- *(parser)* Replace `Crystal::Select::When` with `Crystal::When` ([#14497], thanks @HertzDevil)
+
+[#14442]: https://github.com/crystal-lang/crystal/pull/14442
+[#14497]: https://github.com/crystal-lang/crystal/pull/14497
+
+### Documentation
+
+#### stdlib
+
+- *(collection)* Fix doc for `Set#proper_superset_of?` ([#14516], thanks @meatball133)
+- *(collection)* Fix result formatting in code example for `Indexable#[]?` ([#14721], thanks @meatball133)
+- *(concurrency)* Fix example for `WeakRef` by removing `ref.value` call ([#10846], thanks @hugopl)
+- *(networking)* Improve API docs for `Socket#send` ([#14638], thanks @straight-shoota)
+- *(networking)* Add documentation for `HTTP::WebSocket#stream` ([#14537], thanks @meatball133)
+- *(numeric)* Add documentation for complex methods inside Number ([#14538], thanks @meatball133)
+- *(runtime)* Add documentation for standard streams blocking behaviour ([#14577], thanks @straight-shoota)
+- *(serialization)* Fix docs for `CSV::Builder#row(&)` ([#14736], thanks @philipp-classen)
+- *(system)* **[breaking]** Undocument `IO::Evented` ([#14749], thanks @straight-shoota)
+- *(text)* Fix docs for `Char#ascii_number?` stating wrong minium base ([#14521], thanks @meatball133)
+- *(text)* Enhance documentation for regex options `NO_UTF_CHECK` ([#14542], thanks @straight-shoota)
+
+[#14516]: https://github.com/crystal-lang/crystal/pull/14516
+[#14721]: https://github.com/crystal-lang/crystal/pull/14721
+[#10846]: https://github.com/crystal-lang/crystal/pull/10846
+[#14638]: https://github.com/crystal-lang/crystal/pull/14638
+[#14537]: https://github.com/crystal-lang/crystal/pull/14537
+[#14538]: https://github.com/crystal-lang/crystal/pull/14538
+[#14577]: https://github.com/crystal-lang/crystal/pull/14577
+[#14736]: https://github.com/crystal-lang/crystal/pull/14736
+[#14749]: https://github.com/crystal-lang/crystal/pull/14749
+[#14521]: https://github.com/crystal-lang/crystal/pull/14521
+[#14542]: https://github.com/crystal-lang/crystal/pull/14542
+
+### Specs
+
+#### stdlib
+
+- Remove incorrect uses of `describe` ([#14757], thanks @HertzDevil)
+- *(runtime)* Add specs for `Pointer::Appender` ([#14719], thanks @straight-shoota)
+- *(text)* Add test helper for `normalize/regex_spec` ([#14545], thanks @straight-shoota)
+
+[#14757]: https://github.com/crystal-lang/crystal/pull/14757
+[#14719]: https://github.com/crystal-lang/crystal/pull/14719
+[#14545]: https://github.com/crystal-lang/crystal/pull/14545
+
+### Infrastructure
+
+- *(interpreter)* Run `primitives_spec` with the interpreter on CI ([#14438], thanks @HertzDevil)
+
+[#14438]: https://github.com/crystal-lang/crystal/pull/14438
```

Comment:
This should be moved to the end

Reply:
but why isn't it reflected here still?

Reply:
I hadn't regenerated the changelog since merging that PR.

Reply:
ah! sorry for the noise then!